### ISSUE TITLE
How to disable warning from list_libraries()?

### ISSUE THREAD

**Issue by rtadewald** [man-group/arctic#872]
#### Arctic Version

```
1.79.3
```

#### Arctic Store

```
All of them
```

#### Description of problem and/or code sample that reproduces the issue

How can I disable the warning from `store.list_libraries()`?

`WARNING:root:Could not read from cache due to: not authorized on meta_db to execute command { find: "cache", filter: { type: "list_libraries" }, limit: 1, singleBatch: true, lsid: { id: UUID("68522509-d7cb-46b5-aad6-2ce56242914a") }, $db: "meta_db", $readPreference: { mode: "primaryPreferred" } }. Ask your admin to give read permissions on meta_db:cache`

**Comment by shashank88** [comment]
Can you grant yourself read perms on that db?

**Comment by rtadewald** [comment]
Hi, @shashank88. Thanks for the answer. 

I've already tried but I got a strange issue.
I have 3 different arctic databases (each of them with it's own user).
![image](https://user-images.githubusercontent.com/22618324/95362101-fc44f080-08a3-11eb-8a7b-0b00583e571b.png)

I granted permission to the users, using:
```
use "db"
db.getSiblingDB('db').grantRolesToUser("user", [{role: "readWrite", db: "meta_db"}])
```

But, when I did that, the command  `list_libraries()` started to show (sometimes) the other databases's libraries. I think it's related to the cache, but I dunno how to disable it. 

That's why I decided to revoke that permission and ask how to disable the warning.
